desmume/src/frontend/modules/osd/agg/agg_osd.cpp

```diff
@@ -624,9 +709,11 @@ OSDCLASS::OSDCLASS(u8 core)
 
 	singleScreen = false;
 	swapScreens = false;
-
+	scale=1;
```

Comment:
good idea to make it an int here, when it's a float elsewhere ? (even if currently fractions cause graphics glitches, but that might be fixed at some point)

Reply:
Good point, I'll change to double.